api/services/FootprintService.js

```diff
@@ -181,16 +181,26 @@ module.exports = class FootprintService extends Service {
    * @return Promise
    */
   createAssociation(parentModelName, parentId, childAttributeName, values, options) {
-    /*
-     const parentModel = this.app.orm[parentModelName] || this.app.packs.sequelize.orm[parentModelName]
-     const childAttribute = parentModel.associations[childAttributeName]
-     const childModelName = childAttribute.target.name
-
-     values[childAttribute.foreignKey] = parentId
+    const parentModel = this.app.orm[parentModelName] || this.app.packs.sequelize.orm[parentModelName]
+    const association = parentModel.associations[childAttributeName]
+    const childModelName = association.target.name
 
-     return this.create(childModelName, values, options)
-     */
-    return Promise.reject('trailpack-sequelize does not have createAssociation support yet. Sorry')
+    if (!association) {
+      return Promise.reject(new Error('No such association ' + childAttributeName + ' found'))
```

Comment:
Can you use same error as here please ? https://github.com/trailsjs/trailpack-sequelize/blob/master/api/services/FootprintService.js#L26 like this controller know it have to return a 404 with witch message. 
Thanks